plugins/polio/js/src/domains/Calendar/campaignCalendar/Body.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from 'react';
+
+import { TableRow, TableBody } from '@mui/material';
+import { useStyles } from './Styles';
+
+import { getCells } from './utils';
+import { StaticFieldsCells } from './cells/StaticFields';
+import { PlaceholderRow } from './PlaceholderRow';
+import { RoundPopperContextProvider } from './contexts/RoundPopperContext';
+import { CalendarData, MappedCampaign } from './types';
+
+type BodyProps = {
+    campaigns: MappedCampaign[];
+    currentWeekIndex: number;
+    firstMonday: CalendarData['firstMonday'];
+    lastSunday: CalendarData['lastSunday'];
+    loadingCampaigns: boolean;
+    isPdf: boolean;
+};
+
+export const Body: FunctionComponent<BodyProps> = ({
+    campaigns,
+    currentWeekIndex,
+    firstMonday,
+    lastSunday,
+    loadingCampaigns,
+    isPdf,
+}) => {
+    const classes = useStyles();
+    return (
+        <RoundPopperContextProvider>
+            <TableBody>
+                {campaigns.length === 0 && (
+                    <PlaceholderRow loadingCampaigns={loadingCampaigns} />
+                )}
+                {campaigns.map(campaign => (
```

Comment:
```suggestion
                {campaigns.map((campaign :MappedCampaign) => (
```
Doesn' t cost much, but could save checking the props while debugging. (especially since we have so many variations on Campaign, between here and LQAS/IM)